src/bin-install.ts

```diff
@@ -14,15 +14,21 @@ typings install [<name>=]<location>
   <location>  The location to read from (described below)
 
 Valid Locations:
-  [<source>!]<pkg>[@<version>][#<tag>]
+  <pkg>[@<version>][#<tag>]
+    Install from "defaultSource" specified in `.typingsrc`
+      default to "npm" for non-ambient package
+      default to "dt" for ambient package
+  <source>!<pkg>[@<version>][#<tag>]
+    Install from registry mirror or DefinitelyTyped (see <source> below)
```

Comment:
Coming from TSD, not DT - let's make sure that's clear. People are used to something that only works for one repo, because that's all TSD did - install from DefinitelyTyped. I'd prefer we improve docs, I can copy information into the FAQ from my original issues like https://github.com/typings/typings/issues/72.